kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/asset/provider/BaseAsset.java

```diff
@@ -555,9 +580,14 @@ public void onDriverEvent(final DriverEvent event) {
         final Set<String> channelPrefixes = CollectionUtil.newHashSet();
         for (final Map.Entry<Long, Channel> entry : channels.entrySet()) {
             final Long key = entry.getKey();
-            final String prefix = key + CHANNEL_PROPERTY_POSTFIX.value() + CHANNEL_PROPERTY_PREFIX.value()
-                    + CHANNEL_PROPERTY_POSTFIX.value();
-            channelPrefixes.add(prefix);
+
+            final StringBuilder channelPrefix = new StringBuilder();
```

Comment:
Good source of knowledge but due to static concatenation, StringBuilder and String concatenation works equally. For dynamic concatenation, I completely agree that StringBuilder performs better than String concatenation.